### PR TITLE
[#166896717,#166896762,#166896818] Restyle profile home screen

### PR DESCRIPTION
**Pull request by valentinacala**
The pr introduces new items:
- the DarkLayout component to display the profile section as represented below. Being some elements common to the existing WalletLayout component, this is updated so that it refers to DarkLayout component too
- the FiscalCodeSCreen to display the fiscal code fac-simile
- the FiscalCodeLandscapeModal to display the landscape version of fiscal code fac-simile 

The pr applies changes for bug fixing:
- on ScreenContent, its bouncing behavior is now accessible by a component attribute
- on ScreenContent, a bug is fixed to set the background to dark gray when the dark attribute is true
- on  jest-e2e.config.js and jest.config.js to solve bug on ts transformed patterns
- on FiscalCodeComponent to solve wrong proportions for content position

Here the final result (list design is reviewed within https://github.com/teamdigitale/io-app/pull/1175):
NewProfileSection

Here the final result on iPhone SE, iPhone X and Android devices:
NewProfileHome2

The contextual help from profile home screen:
image

The contextual help from the fiscal code screen:
image

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#166896717](https://www.pivotaltracker.com/story/show/166896717): Come CIT voglio vedere la home della sezione profilo con il nuovo restyle

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/components/FiscalCodeLandscapeModal.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+});
+
+export default class FiscalCodeLandscapeModal extends React.PureComponent<
+  Props
+> {
+  private contentRef: any;
+
+  public render() {
+    return (
+      <Modal
```
Comment (fpersico) [inline ts/components/FiscalCodeLandscapeModal.tsx]: The only react-native Modal must be the IdentificationModal for all the other "modals" use the LightModal component (withLightModal HOC).

File: ts/screens/profile/FiscalCodeScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            profile={this.props.profile}
+          />
+        )}
+      </React.Fragment>
+    );
+  }
+}
+
+const mapStateToProps = (state: GlobalState) => ({
+  profile: getOrElse(profileSelector(state), undefined)
```
Comment (fpersico) [inline ts/screens/profile/FiscalCodeScreen.tsx]: I think we also have `toUndefined` in the pot library.

Comment (valentinacala) [reply ts/screens/profile/FiscalCodeScreen.tsx]: updated within https://github.com/teamdigitale/io-app/pull/1191/commits/1d38dc8d1a511d43e19b00d574aec0ca29a636c0

File: ts/components/FiscalCodeLandscapeModal.tsx (hunk 1 above)
Comment (valentinacala) [reply ts/components/FiscalCodeLandscapeModal.tsx]: can I propose you to open a new story that includes the migration of components that now use Modal to the LightModal (AbortOnboardingModal, ContextualHelp, FiscalCodeLandscapeModal, ConfirmCardDetailsScreen)? cc @fpersico

**Pull request merged**